kano_updater/utils.py

```diff
@@ -577,3 +579,15 @@ def verify_kit_is_plugged():
             ).run()
 
     return is_plugged and not is_battery_
+
+
+def detach_from_tty():
+    """
+    Detach from controlling tty
+    """
+    try:
+        myTTY = os.open("/dev/tty",os.O_RDWR)
```

Comment:
Should this be broadly better if queried dynamically? To support docker interactive builds.

```
kanux@minos:~$ peldins-jessie
Started container: peldins-jessie with command: 'bash'
update-binfmts: warning: qemu-arm already enabled in kernel.
kanux@4f73998cc87b:~/peldins-jessie$ tty
/dev/pts/12
```

Reply:
`/dev/tty` is a magic file which represents the tty attached to the current process.

Reply:
Enlightning... 👍